lib/index.js

```diff
@@ -208,10 +208,7 @@ internals.graphqlHandler = async function (request, h) {
         ]);
       });
 
-    if (tracingResult.tracingResolvers.length) {
```

Comment:
The only thing that I'm afraid of removing this validation, is that while using `GraphiQL` when you load it the first time or refresh it, there is a request so, it'll display on console an empty table.